pkg/handlers/v1/notification.go

```diff
@@ -65,6 +69,7 @@ func (h *NotificationHandler) Handle(ctx context.Context) (Output, error) {
 		if err := h.TimestampStorer.StoreTimestamp(ctx, scan.Timestamp); err != nil {
 			return Output{}, err
 		}
+		stater.Timing("scannotificationdelay", time.Since(scanCompletedTime)*time.Millisecond)
```

Comment:
question: What is this timer supposed to indicate? As in, what aspect of the system health does it uniquely describe that other metrics don't cover?

Reply:
This metric measures the time between when a site is finished scanning in Nexpose, and when Scan-Notifier sends the site to be processed by the next microservice, Asset-Producer. Because Scan-Notifier polls Nexpose, we want to minimize retrieval of a scanned site, that way we can immediately process assets.